recipe/meta.yaml

```diff
@@ -39,21 +38,18 @@ requirements:
     - python                                # [build_platform != target_platform]
     - pip                                   # [build_platform != target_platform]
     - cross-python_{{ target_platform }}    # [build_platform != target_platform]
-    - numpy 1.26.*                          # [build_platform != target_platform]
-    - qt6-main >=6.6.1,<7.0.0               # [build_platform != target_platform]
+    - qt6-main                              # [build_platform != target_platform]
   host:
     - python
-    - numpy 1.26.*
```

Comment:
Ah can see NumPy was dropped from build dependencies in the in the recent release ( https://github.com/nion-software/nionui-tool/commit/6bdb6b4ca88fde7faf75805ce5b76e1780ec2b52 ). Nice! That will simplify things here